src/docs/sdk/features.mdx

```diff
@@ -105,6 +105,23 @@ Envelope item:
  * [Java](https://github.com/getsentry/sentry-java/blob/b2e8dd43ea2ef37201b215ec409655e49e3231c8/sentry/src/main/java/io/sentry/SentryEnvelopeItem.java#L131-L155)
  * [C#](https://github.com/getsentry/sentry-dotnet/blob/3bc6c66a14e653982a6a25df033d68d8ad12c4af/src/Sentry/Protocol/Envelopes/EnvelopeItem.cs#L170-L178)
  
+## Attachments
+
+Attachments are files stored alongside an event or transaction. To send an attachment, add it as an [envelope item](/sdk/envelopes/#attachment)
+to the corresponding event or transaction.
+
+We recommend implementing two types of attachments, one with a path and another with a byte array.
+If the programming language allows it, create one class with multiple constructors to keep things simple and guess the content type
+of the attachment via the filename. Use the implementations of [Java](https://github.com/getsentry/sentry-java/blob/main/sentry/src/main/java/io/sentry/Attachment.java),
+[Objective-C](https://github.com/getsentry/sentry-cocoa/blob/master/Sources/Sentry/SentryAttachment.m), or
+[Python](https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/attachments.py) as a reference for the API.
+
```

Comment:
```suggestion

The overload that takes a `path` should consider:
* Read the file at the time an event gets captured, and not at the time the attachment is added to the scope.
* If reading the attachment fails, fail gracefully (don't drop the whole envelope)
* If the SDK is in debug mode (`debug=true`) log out errors to help debugging
```

Reply:
Thanks for the input. I slightly modified it.